install.sh

```diff
@@ -17,11 +17,18 @@ then
         exit 1
     fi
 else
-    echo "Node detected"
+    DETECTED="$(node -v)"
+    echo $DETECTED 
```

Comment:
```suggestion
echo "Detected version: $DETECTED"
```

Reply:
Not suitable, the version finished an existing line ` blabla ... v1.1.1`

Reply:
I see.